schemaless/sources.py

```diff
@@ -651,6 +651,26 @@ class AffordableRentalPortfolio(DirectSource):
     DATA_SF = "https://data.sfgov.org/Housing-and-Buildings/Mayor-s-Office-of-Housing-and-Community-Developmen/9rdx-httc"  # NOQA
 
 
+class DAInfo(DirectSource):
+    NAME = 'da_info'
+    OUTPUT_NAME = NAME
+    FK = PrimaryKey(
+        NAME, 'row_number', 'delivery_agency', 'project_name', 'phase')
+    FIELDS = {
+        'Row No.': 'row_number',
+        'Housing Delivery Agency': 'delivery_agency',
+        'Project Name:': 'project_name',
+        'Phase/ Bldg/Address/B+L': 'phase',
```

Comment:
Nit: It's super verbose, but thoughts on naming this 'phase_bldg_address_blklot' so it's closer to what it actually is?